src/node.rs

```diff
@@ -550,6 +550,18 @@ impl<'a> NodeProperty<'a> {
         core::str::from_utf8(self.value).map(|s| s.trim_end_matches('\0')).ok()
     }
 
+    /// Attempts to parse the property value as a list of [`&str`].
+    pub fn as_str_list(self) -> impl Iterator<Item = &'a str> {
+        let mut start = 0;
+
+        core::iter::from_fn(move || {
```

Comment:
This stops after the first value, and stops updating the `start` variable.

Not sure if there is an actual way to get this to work as intended. I think it is failing because the `value` slice is consumed on the first call, becoming zero-length on the second iterator call.